src/uu/seq/src/seq.rs

```diff
@@ -211,16 +121,38 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
                 false,
             )
         }
+        (Number::MinusZeroFloat, last, increment) => print_seq(
```

Comment:
This should also cover `MinusZeroInt`

Reply:
Yes, I'll add a test case for that as well. Thanks.